src/templates/en/2019/base_chapter.html

```diff
@@ -0,0 +1,5 @@
+{% extends "en/base.html" %}
+
+{% block title %}
+Chapter {{ chapter }} | The Web Almanac by HTTP Archive
```

Comment:
Use the chapter metadata.

```suggestion
{{ metadata.get('title') }} | 2019 | The Web Almanac by HTTP Archive
```